azure-pipelines.yml

```diff
@@ -57,7 +73,13 @@ steps:
   inputs:
     workingDir: 'build'
     cmakeArgs: '$(CMAKE_EXTRA_FLAGS) ../src'
-  condition: eq( variables['Agent.OS'], 'Darwin')
+  condition: or( eq( variables['Agent.OS'], 'Windows_NT'), eq( variables['Agent.OS'], 'Darwin'))
+- task: VSBuild@1
+  inputs:
+    solution: 'build/*.sln'
+    vsVersion: 'latest'
+  condition: eq( variables['Agent.OS'], 'Windows_NT')
+  displayName: 'Visual Studio Build'
 - script: |
     cd build
     make
```

Comment:
Note for future -- it looks like we're building on windows, but not yet testing.  We should fix that in a separate PR.

Tag: @agcapps

Reply:
Tag noted.